### PR TITLE
Rename upcoming deprecated Lifecycle methods

### PR DESCRIPTION
**Pull request by ekh64**
`componentWillReceiveProps` => `UNSAFE_componentWillReceiveProps`
`componentWillUpdate` => `UNSAFE_componentWillUpdate`

Apparently in React 17.x these will continue to work 🎉 but also they're still unsafe 😒 

addresses #181

### CODE REVIEW

File: src/LineChart.js [hunk 1]
```diff
@@ -17,32 +17,25 @@ export default class LineChart extends React.Component {
...
     /**
      * Accessor function for line X values, called once per datum, or a single value to be used for the entire line.
-     */
-    x: CustomPropTypes.valueOrAccessor,
+     */ x: CustomPropTypes.valueOrAccessor,
     /**
      * Accessor function for line Y values, called once per datum, or a single value to be used for the entire line.
-     */
-    y: CustomPropTypes.valueOrAccessor,
+     */ y: CustomPropTypes.valueOrAccessor,
```
Comment (acmei) [inline src/LineChart.js]: Is your editor removing this space for some reason?

File: src/LineChart.js [hunk 2]
```diff
@@ -17,32 +17,25 @@ export default class LineChart extends React.Component {
     data: PropTypes.array.isRequired,
     /**
      * Accessor function for line X values, called once per datum, or a single value to be used for the entire line.
-     */
-    x: CustomPropTypes.valueOrAccessor,
+     */ x: CustomPropTypes.valueOrAccessor,
```
Comment (krissalvador27) [inline src/LineChart.js]: The formatting looks like it went rogue here and is repeated a few times below and in other files 😅

Comment (ekh64) [reply src/LineChart.js]: I'm so confused because vscode should pick up the prettier config 😕

File: src/LineChart.js (hunk 1 above)
Comment (ekh64) [reply src/LineChart.js]: I'm using vscode which does prettier on save. Lemme deal with that

**Comment by ekh64** [comment]
OK this should resolve the weirdness with the comments 🤷‍♀

**Comment by ekh64** [comment]
@krissalvador27 I updated this, lmk if we can merge 🙏

**Pull request merged**